build-system/exec.js

```diff
@@ -28,8 +28,8 @@ const shellFlag = (process.platform == 'win32') ? '/C' : '-c';
  * Spawns the given command in a child process with the given options.
  *
  * @param {string} cmd
- * @param {Object} options
- * @return {Object} Process info.
+ * @param {object} options
```

Comment:
AMP's preferred annotation appears to be `Object` (upper case).

The lint plugin error can be overridden [like this](https://github.com/ampproject/amphtml/blob/c3a0d7c/.eslintrc#L89) (#22498):

```
"jsdoc/check-types": [2, { "noDefaults": true }],
```

Reply:
I was thinking of leaving this PR with more "defaults" compared to `ampproject/amphtml` repo, which earned the right to be opinionated. I don't feel strongly about it one way or the other, but if you do I'll introduce this override. Let me know